reposerver/src/main/resources/application.conf

```diff
@@ -17,6 +17,7 @@ storage {
     bucketId = ${?TUF_REPOSERVER_AWS_BUCKET_ID}
     region = "eu-central-1"
     region = ${?TUF_REPOSERVER_AWS_REGION}
+    endpointUrl = ${?TUF_REPOSERVER_S3_URL}
```

Comment:
this needs an extra line:

```
endpointUrl = ""
```

Otherwise wont run without `TUF_REPOSERVER_S3_URL` set.